### PR TITLE
create a template for RAG Dev Bootcamp 2024

### PR DESCRIPTION
**Pull request by marwan116**

### CODE REVIEW

**Comment by marwan116** [comment]
Note to self:
Seems like after we merge this, I will need to do this step:

- Update the product repo backend/workspace-templates.yaml to point to the new template added here after being merged.

would have been nice if pipeline handled it for me automatically ...

**Review by sofianhnaide — APPROVED** [review]
LGTM

**Comment by sofianhnaide** [comment]
I will follow up to fix the CI and prevent merging if it is busted

**Comment by gvspraveen** [comment]
fix lint in master. please rebase

File: configs/rag-dev-bootcamp-mar2024/aws.yaml
Comment (gvspraveen) [inline configs/rag-dev-bootcamp-mar2024/aws.yaml:4]: Do we need this if we are using serverless configs?

See endpoints OA template: https://github.com/anyscale/templates/blob/main/configs/endpoints_v2/aws.yaml

cc @shomilj @sofianhnaide

**Comment by marwan116** [comment]
> fix lint in master. please rebase

@gvspraveen is this blocking merge from your perspective?

**Comment by marwan116** [comment]
> > fix lint in master. please rebase
> 
> @gvspraveen is this blocking merge from your perspective?

Well nevermind - I just rebased

**Pull request merged**
